python/src/equistore/operations/ones_like.py

```diff
@@ -20,6 +21,119 @@ def ones_like(
                        If empty list ``[]`` no gradients information are copied.
     :param requires_grad: If autograd should record operations for the returned tensor.
                           This option is only relevant for torch.
+
+    Here is an example using this function:
+
+    First we create a ``TensorMap`` with just one block with two gradients, named ``alpha`` and `beta`.
+
```

Comment:
and add the import here
```suggestion
    >>> import numpy as np
    >>> from equistore import Labels
```